### PR TITLE
Feat: Card Markup: Titles

### PR DESCRIPTION
**Pull request by catheraaine**
Adds styles for card, card title, and some fonts.

### CODE REVIEW

**Comment by BC3209** [comment]
I think it looks good. Anyone else have any input @iAmNathanJ @boomingbrake

File: app/assets/stylesheets/events.scss [hunk 1]
```diff
@@ -1,3 +1,65 @@
...
+}
+
+// VARIABLES
+
+$light: #ffffff;
+$dark-gray: #292929;
+$mid-gray: #4A4A4A;
+$light-gray: #9B9B9B;
+$highlight: #F9ED00;
+$border-opacity: rgba(155, 155, 155, .3);
```
Comment (iAmNathanJ) [inline app/assets/stylesheets/events.scss]: I think this should be renamed to `$border-color` as it is a color. I think everything else looks great.

**Comment by catheraaine** [comment]
@BC3209 @iAmNathanJ @boomingbrake

**Comment by BC3209** [comment]
hmmmm looking good! 


Anyone else want to take a look at this? @iAmNathanJ @boomingbrake

**Comment by boomingbrake** [comment]
Looking good!

**Pull request merged**